test-quickstart-template.groovy

```diff
@@ -55,6 +56,8 @@ node('quickstart-template') {
 
                     dir('testing/citest') {
                         sh activate_venv + 'pip install -r requirements.txt;PYTHONPATH=.:spinnaker python tests/kube_smoke_test.py --native_host=localhost --spinnaker_kubernetes_account=my-kubernetes-account'
+
+                        archiveArtifacts 'kube_smoke_test.*'
```

Comment:
If you need you can also upload the artifacts to Azure Storage:
https://github.com/azure-devops/ci/blob/master/vars/buildPlugin.groovy#L65